tests/monitoring/monitoring.go

```diff
@@ -238,3 +248,11 @@ func checkRequiredLabels(rule promv1.Rule) {
 	ExpectWithOffset(1, rule.Labels).To(HaveKeyWithValue("kubernetes_operator_component", "kubevirt"),
 		"%s kubernetes_operator_component label is missing or not valid", rule.Alert)
 }
+
+func requirePrometheusRuleCRD(virtClient kubecli.KubevirtClient) {
```

Comment:
@avlitman Please coordinate with #16318, I think its better to replace skips with failures.